ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModule.java

```diff
@@ -83,6 +83,14 @@ public UIManagerModule(
     reactContext.addLifecycleEventListener(this);
   }
 
+  /**
+   * This method gives an access to the underlying native view hierarchy manager. Note that all the
+   * operations executed on that object should be run in the UI thread. Use with caution
+   */
+  public NativeViewHierarchyManager getNativeViewHierarchyManager() {
+    return mUIImplementation.getUIViewOperationQueue().getNativeViewHierarchyManager();
```

Comment:
We should probably not include rebound but instead just inline the spring implementation like I did: https://github.com/facebook/react-native/blob/master/Libraries/Animated/src/AnimatedImplementation.js#L302-L514

Rebound comes with a bunch of utilities and classes that we don't need for Animated. The core "physics" is actually just a loop with only a dozen of lines of code.